extra/2-piping.js

```diff
@@ -16,26 +16,35 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(numberOne, numberTwo) {
+  return numberOne + numberTwo;
 }
 
-function multiply() {
-
+function multiply(numberOne, numberTwo) {
+  return numberOne * numberTwo;
 }
 
-function format() {
-
+function format(lonelyNumber) {
+  return `£${lonelyNumber}`;
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+
+// So, I'm not too sure what bad code or good code looks like yet so I wrote this, it seems vague and repetitive:
+// let badCode = add(10, startingValue); badCode = multiply(badCode, 2); badCode = format(badCode);
+
+// Below, I wrote some more of what I interpret bad code to look like too. This way seems complicated for me to read functions nested in others (although not sure if that's down to me being a beginner!):
+let badCode = format(multiply(add(10, startingValue), 2));
```

Comment:
Hi Mandy, I think you did a great job, exactly what was supposed to be done in this exercise. 
It's definitely not you being a beginner, I don't think it would be easy to understand this code for anybody, no matter what level of knowledge they had.

Reply:
Hi Paulina, thank you for reviewing my code and for your reassuring comments :)